packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2069,6 +2073,303 @@ export const audiusBackend = ({
     }
   }
 
+  function getDiscoveryEntityType(type: string) {
+    if (type === 'track') {
+      return Entity.Track
+    }
+    return Entity.Playlist
+  }
+
+  function formatBaseNotification(notification: DiscoveryNotification) {
+    const timestamp = notification.actions[0].timestamp
+    return {
+      groupId: notification.group_id,
+      timestamp: new Date(timestamp * 1000).toString(),
+      isViewed: notification.seen_at == null,
+      id: `timestamp:${timestamp}:group_id:${notification.group_id}`
+    }
+  }
+
+  function mapDiscoveryNotification(notification: DiscoveryNotification) {
```

Comment:
this all totally makes sense, but long term are you thinking we move our data types to be in line with discovery-notification data?

Reply:
yea fo sho
was just mapping the new notifs to the old as the easiest way to get this done without potentially breaking stuff